client/build-config/src/paths.ts

```diff
@@ -1,8 +1,22 @@
+import { realpathSync } from 'fs'
 import path from 'path'
 
-export const ROOT_PATH = path.resolve(__dirname, '../../../')
-export const WORKSPACES_PATH = path.resolve(ROOT_PATH, 'client')
-export const NODE_MODULES_PATH = path.resolve(ROOT_PATH, 'node_modules')
-export const MONACO_EDITOR_PATH = path.resolve(NODE_MODULES_PATH, 'monaco-editor')
-export const STATIC_ASSETS_PATH = path.resolve(ROOT_PATH, 'ui/assets')
-export const STATIC_INDEX_PATH = path.resolve(STATIC_ASSETS_PATH, 'index.html')
+// NOTE: use fs.realpathSync() in addition to path.resolve() to resolve
+// symlinks to the real path. This is required for webpack plugins using
+// `include: [...file path...]`.
```

Comment:
Should we mention how this is related to using `pnpm`?

Reply:
I'm still not 100% sure if this is the greatest idea. I worry we'll need fixes like this all over the place.

Would be better if we found a fix specifically to help webpack work with symlinks, instead of us having to do things like this to workaround webpack not working with symlinks...

Reply:
@gregmagolan Wdyt?